rotkehlchen/chain/evm/utils.py

```diff
@@ -171,3 +180,28 @@ def lp_price_from_uniswaplike_pool_contract(
     numerator = token0_supply * token0_price + token1_supply * token1_price
     share_value = numerator / total_supply
     return Price(share_value)
+
+
+def maybe_notify_cache_query_status(
+        msg_aggregator: 'MessagesAggregator',
+        last_notified_ts: Timestamp,
+        protocol: str,
+        chain_name: str,
```

Comment:
why pass chain name and not either ChainID or Blockchain enum here? It's best to do conversion/serialization when stuff leave our code. So right here.

This is informally called the sandwich pattern. Decide on a format for your codebase. For us it's these enums. And code that comes from the outside can be anything but needs to be serialized into it at point of entry (API, CSV import etc) and deserialized to it at point of exit (such as these websocket messages).